amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/ActivityImporter.java

```diff
@@ -237,9 +235,9 @@ private void init(JsonBean newJson, boolean update, String endpointContextPath)
             validateAndImport(newActivity, newJsonParent);
             if (errors.isEmpty()) {
                 prepareToSave();
-                boolean draftChange = ActivityUtil.detectDraftChange(newActivity, oldActivityDraft);
```

Comment:
I am afraid this will be a regression: before `draftChange` for API was calculated based on `oldActivityDraft`. Please restore.

Reply:
Indeed, you are right. I still get confused about why this regression appeared.